koiic/vehicle-store-api
# #3 An endpoint to retrieve a single car object

**What does this PR do?**

An endpoint to retrieve a single car object

**Description of Task to be completed?**

As a user, I should be able to retrieve a single car endpoint

**How should this be manually tested?**

- Fetch the branch with `git fetch origin ft-fetch-single-car-03 `
- Check out the branch with `git checkout ft-fetch-single-car-03 `
- Run the app: `docker-compose up` or `python manage.py runserver`
- On postman create a get request to this url `http://127.0.0.1:8000/api/car/getone/<int:id>/`

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**

N/A
